src/renderer/mod.rs

```diff
@@ -771,7 +771,16 @@ impl<'a> RenderApi<'a> {
                     fg: Color::Spec(Rgb { r: 0, g: 0, b: 0}),
                     flags: cell::Flags::empty(),
                 }
-            })
+            }, IndexedCell {
```

Comment:
It's because I return, tuple from `RenderableCellsIter::next` and `render_cells` takes tuple too.